### DISCUSSION TITLE
Approximate Release Date for TShock built on Terraria 1.4.4

### DISCUSSION THREAD

**Discussion by EMREOYUN** [Pryaxis/TShock#2667] [General]
Today, Terraria 1.4.4 was released. When a new TShock version for 1.4.4 will be released?

**Comment by punchready** [comment]
this update is pretty big so it might take a while for a stable release

**Reply by chairclr** [reply]
hi punchready

**Comment by ghost** [comment]
how long does this normally take have made my own terraria server and have a butch of stuff on and trying to grow it how long did this take for the last update?

**Comment by TheCToTheY** [comment]
It's 1.4.4.1 already... The final final final final final update. For sure.

**Reply by TheCToTheY** [reply]
Nvm, 1.4.4.2.

**Reply by ghost** [reply]
Lol

**Reply by PTALTS-LK** [reply]
*journey's end*

**Reply by hakusaro** [reply]
I think there are still substantial bugs in 1.4.4.2 so we may get another hotfix, based on our understanding of some of the new network code and mechanics.

**Comment by ghost** [comment]
Ehen will a unstable release be out. Well I probably should wait for a week so it's more stable release. But where can I get them

**Reply by hakusaro** [reply]
There are video instructions on discord for unstable releases; unstable builds of some variety are available. Re-Logic caught us off guard in the middle of a lot of important update work, so we had to undo some of our work temporarily and we’re still recovering from that. This has delayed what would normally be a fast update.

**Reply by BlueMpoopSir** [reply]
very excited for the update when it is released, god speed. but take your time no rush lol

**Reply by SMH-ImCold** [reply]
Haha can't wait
